### ISSUE TITLE
How to get pretrained model and what is it trained on?

### ISSUE THREAD

**Issue by Vibha111094** [AlvaroCavalcante/auto_annotate#37]
Hi @AlvaroCavalcante ,
I am trying to understand the logic behind the automatic annotation.
Is the concept as follows:
-you label a few instances of data for your use case.
-Use this data to train a pre-trained object detection model.
-Then use the fine-tuned model to make bounding box inference on new data?

**Comment by AlvaroCavalcante** [comment]
The second situation is when you have a large set of images. Imagine that you label manually like 500 images and creates a model to test the convergence and how the subset of data performs on the model. In this initial test, you'll have a pre-trained model that could be helpful to automatically label the remaining images of your dataset! 

If you have any doubt, let me know! I also have a tutorial on Medium that you can read: https://medium.com/analytics-vidhya/auto-annotate-images-for-tensorflow-object-detection-19b59f31c4d9

**Comment by Vibha111094** [comment]
Thankyou @AlvaroCavalcante. This helps!
